test/extended/builds/run_fs_verification.go

```diff
@@ -0,0 +1,103 @@
+package builds
+
+import (
+	"fmt"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/image"
+)
+
+var _ = g.Describe("[sig-builds][Feature:Builds] verify /run filesystem contents", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc                  = exutil.NewCLI("verify-run-fs")
+		testBuildConfigYaml = fmt.Sprintf(`
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: verify-run-fs
+spec:
+  runPolicy: Serial
+  source:
+    dockerfile: "FROM %s\nRUN ls -R /run\nUSER 1001"
+    type: Dockerfile
+  strategy:
+    dockerStrategy:
+      env:
+      - name: BUILD_LOGLEVEL
+        value: "10"
+      imageOptimizationPolicy: SkipLayers
+    type: Docker
+`, image.ShellImage())
+		lsRSlashRun = `
+/run:
+lock
+rhsm
+secrets
+
+/run/lock:
+
+/run/rhsm:
+
+/run/secrets:
+kubernetes.io
+openshift.io
```

Comment:
ultimately the goal is that these 2 dirs should not show up.  So i think the test is fine, but let's hold it until we can land it w/ the correct set of expected dirs.

Reply:
agreed

/hold

Reply:
do you want to at least approve @bparees so someone else can do final review and lgtm when we are ready ? 

or would you prefer to be the one that gates merging?